lib/services/spatial.js

```diff
@@ -21,25 +21,30 @@ class SpatialService {
     this._zk = zookeeper;
   }
 
+  _destroyLayers(fourfours, core, cb) {
+    return async.map(fourfours, core.destroy.bind(core), (err, destroyResponses) => {
```

Comment:
`core.destroy.bind(core)`? How does that work? Doesn't `destroy` take 2 arguments?

Javascript :cry: